jplag/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This is for static mocks ?

Reply:
for final classes. I think it was necessary to mock an instance of a record